source/nijigenerate/viewport/model/package.d

```diff
@@ -28,6 +28,19 @@ private {
     enum ENTRY_SIZE = 48;
 }
 
+enum ViewporMenuSortMode {
+    ZSort,
+    SizeSort,
+}
+
+/** 
+    Default to SizeSort because when selecting objects, especially detailed puppets 
+    or small components, we assume the user will point directly at the one they want. 
+    Prioritizing smaller objects makes them easier to select, while larger objects 
+    can usually be selected by clicking elsewhere if needed.
+*/
+ViewporMenuSortMode incViewportModelMenuSortMode = ViewporMenuSortMode.SizeSort;
```

Comment:
It seems ViewportMenuSortMode is not serialized to settings.json yet.
Until serialization is implemented, I want to keep ZSort mode by default.